_pages/fsa/install/building-the-fsa-storefront-from-libraries-1-0.md

```diff
@@ -0,0 +1,163 @@
+
+You should add the following ngrx store module with specified runtime checks option in the imports section:
+
+```
+    StoreModule.forRoot({}, {
+      runtimeChecks: {
+        strictStateImmutability: false,
+        strictActionImmutability: false,
+      },
+    }),
+```
+It also requires ngrx import:
+```
+import { StoreModule } from '@ngrx/store';
+```
+
+After FSA is installed your app.module.ts should look like following:
+```
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { StoreModule } from '@ngrx/store';
+import { AppComponent } from './app.component';
+import { FSStorefrontModule } from '@spartacus/fsa-storefront';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    StoreModule.forRoot({}, {
+      runtimeChecks: {
+        strictStateImmutability: false,
+        strictActionImmutability: false,
+      },
+    }),
+    FSStorefrontModule.withConfig({
+      backend: {
+        occ: {
+          baseUrl: '/--path to the server---/',
+          prefix: '/occ/v2/'
+        }
+      },
+      context: {
+        currency: ['EUR'],
+        language: ['en', 'de'],
+        urlParameters: ['baseSite', 'language', 'currency'],
+        baseSite: ['financial']
+      },
+      authentication: {
+        client_id: 'financial_customer',
+        client_secret: 'secret'
+      },
+      features: {
+        consignmentTracking: true,
+      }
+    })
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+For example, check:
+- `baseUrl`: Points to your SAP Commerce server
+- `prefix`: Defines the prefix to OCC calls; change `/rest/v2/` to `/occ/v2/` if using release 2005
+- `context`: Defines the site context such as base site, language, and currency.
+- `authentication`: Defines authorization of financial customer
+ 
+*Note:* If your setup failed and for some reason your app.module.ts is not configured like described please check one more time requirements for fsa schematics usage, Angular CLI version should be <= 10.0!
+
+### Starting your Spartacus app ###  
+
+Start your app with the following command:
+
+```
+yarn start
+```
+
+Your app will be compiled and then started.
+
+To display your storefront, assuming everything is installed locally, browse to `http://localhost:4200`. If you installed Financial sample data with the financialstorefront extension, the FSA Spartacus storefront for Financial services should appear.
+
+Note: If your storefront doesn't appear, you probably have to accept a privacy certificate. To do so, browse to `https://localhost:9002/occ/v2/electronics/cms/pages`, and then accept the privacy certificate. This step is necessary because your browser will block calls to app which makes calls to localhost:9002, due to security settings. To see the browser message, right-click in your browser, select *Inspect*, then click *Console*.
```

Comment:
`https://localhost:9002/occ/v2/electronics/cms/pages` should be changed to `https://localhost:9002/occ/v2/financial/cms/pages`

Reply:
Fixed.